### PR TITLE
Added automatic exporting GCP cloud to kentik in terraform module

### PR DESCRIPTION
**Pull request by lukaa12**
KNTK-214

### CODE REVIEW

File: cloud_GCP/terraform/module/examples/all-network-subnets/main.tf [hunk 1]
```diff
@@ -1,25 +1,41 @@
...
 data "google_compute_subnetwork" "subnetworks" {
-  count  = "${length(data.google_compute_network.network.subnetworks_self_links)}"
-  name = "${element(split("/", data.google_compute_network.network.subnetworks_self_links[count.index]), 10)}"
+  count  = length(data.google_compute_network.network.subnetworks_self_links)
+  name = element(split("/", data.google_compute_network.network.subnetworks_self_links[count.index]), 10)
 }
 
 module "kentik_gcp_integration" {
-  source = "../../"
+  source             = "/home/lukasz/Projects/config-snippets-cloud/cloud_GCP/terraform/module"
```
Comment (Fenthick) [inline cloud_GCP/terraform/module/examples/all-network-subnets/main.tf:34]: Please keep relative path

**Review by Fenthick — CHANGES_REQUESTED** [review]

File: cloud_GCP/terraform/module/examples/subnet-list/main.tf [hunk 2]
```diff
@@ -1,16 +1,32 @@
...
 provider "google" {
-  version = ">= 3.41.0"
-  credentials = file("${var.credentials}")
+  credentials = file(var.credentials)
   project     = var.project
   region      = var.region
 }
 
+provider "kentik-cloudexport" {
+
```
Comment (mateuszmidor) [inline cloud_GCP/terraform/module/examples/subnet-list/main.tf:21]: maybe it's worthwhile to add comment to cloudexport providers that email and token are read from env variables, for clarity?

**Review by mateuszmidor — APPROVED** [review]

File: cloud_GCP/terraform/module/examples/all-network-subnets/main.tf (hunk 1 above)
Comment (Fenthick) [reply cloud_GCP/terraform/module/examples/all-network-subnets/main.tf]: Resolved

File: cloud_GCP/terraform/module/examples/subnet-list/main.tf (hunk 2 above)
Comment (Fenthick) [reply cloud_GCP/terraform/module/examples/subnet-list/main.tf]: Resolved

**Review by Fenthick — APPROVED** [review]

**Review by mmac-m3a — APPROVED** [review]
LGTM

**Pull request merged**